cmake/ctest/drivers/ascicgpu031/CMakeLists.txt

```diff
@@ -27,11 +27,26 @@ TRILINOS_DRIVER_ADD_DASHBOARD(
 
 TRILINOS_DRIVER_ADD_DASHBOARD(
   DEBUG_CUDA_NO_UVM_CRS
-  ctest_linux_nightly_mpi_release_nouvm_crs_ascicgpu031.cmake
+  ctest_linux_nightly_mpi_debug_nouvm_crs_ascicgpu031.cmake
```

Comment:
seems like the "crs" is in front of "ascicgpu031" in this file, but afterwards in the actual filename